### ISSUE TITLE
[Elasticsearch-Connector] Access inner_hits

### ISSUE THREAD

**Issue by kokolem** [elastic/search-ui#782]
**Is your feature request related to a problem? Please describe.**
When using a nested query:

```js
const connector = new ElasticsearchAPIConnector({
  host: "http://localhost:9200",
  index: "index"
}, (requestBody, requestState) => {
  if (!requestState.searchTerm) return requestBody;

  requestBody.query = {
    nested: {
      path: "nested_object",
      query: {
        multi_match: {
          query: requestState.searchTerm,
          fields: ["field"]
        }
      },
      inner_hits: {}
    }
  }

  return requestBody;
})
```

there is currently no way to access `inner_hits` of the results.

**Describe the solution you'd like**
It would be nice to be able to access inner hits from the `result` parameter provided to `resultView`.

**Describe alternatives you've considered**
Unfortunately I haven't found any way to access inner_hits. Suggestions are welcome.

**Comment by joemcelroy** [comment]
Thanks @kokolem  - an interesting use-case. I dont have a solution or workaround right now. Likely the best solution is providing the raw elasticsearch item hit with the result object so that you can render the nested documents.

I think long term we want to make nested document support 1st class. Facets and query to support nested objects.

**Comment by joemcelroy** [comment]
released in 1.17.0
